src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let partnerMessage = 'I am not married';
+
+  if (person.isMarried) {
+    partnerMessage =
+      person.sex === 'm'
+        ? `${person.partnerName} is my wife`
+        : `${person.partnerName} is my husband`;
```

Comment:
Only the word wife/husband different, so set it conditionally instead of repeating all